### PR TITLE
Hangman

### PR DESCRIPTION
**Pull request by nyron**
Comfort Level: 2/5

### CODE REVIEW

**Comment by nyron** [comment]
With a little help I made some edits to my hangman because I needed to do better. Although it can be improved it is a start and I am challenged but ready for more challenges on my path!

File: script.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+var word;
```
Comment (nolds9) [inline script.js:1]: How could we refactor to use an object-oriented approach? These global variables could be properties on an object

File: script.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  console.log(splitword);
+
+  startgame();
+  win();
+}
+
+function startgame(){
+
+guess = prompt("okay, guess a letter");
+var index = splitword.indexOf(guess);
```
Comment (nolds9) [inline script.js:22]: :+1:  clever

File: script.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+console.log(index);
+if(index != -1) {
+  alert("Correct Letter");
+  splitword.splice(index , 1);
+  console.log(splitword);
```
Comment (nolds9) [inline script.js:27]: Can you think of a function that shows the user the currentState of the game, such as a `displayWord` function that shows the word with the missing letters as underscores. Hint, you could reuse the `index` as a key to "fill in the appropriate blank"

**Comment by nolds9** [comment]
:+1: Nice job on this. If you're interested, please check out the [solution](https://github.com/ga-wdi-exercises/hangman-lab/blob/jesse-review-solution/hangman.js)

**Pull request closed without merging**